iib/web/models.py

```diff
@@ -930,8 +955,17 @@ def from_json(cls, kwargs, batch=None):
             request_kwargs[key] = [
                 Image.get_or_create(pull_specification=item) for item in request_kwargs.get(key, [])
             ]
-
+        build_tags = []
```

Comment:
What about request_kwargs.pop('build_tags', []) instead of those 3 lines?
This logic can be done in Request._from_json() to avoid uplicates